src/services/cipher.service.ts

```diff
@@ -63,6 +63,8 @@ export class CipherService implements CipherServiceAbstraction {
     // tslint:disable-next-line
     _decryptedCipherCache: CipherView[];
 
+    private lastSortedCiphers: CipherView[] = [];
```

Comment:
This should be cleared up when the app is locked, just like `_decryptedCipherCache`, right? See `clearCache()`